projects/js-packages/components/components/action-popover/stories/styles.module.scss

```diff
@@ -0,0 +1,11 @@
+.wrapper {
```

Comment:
For me, looks strange that we're taking the entire screen and not providing some visual feedback that we're doing it (for example a light dark background that regular is used).

This kind of generate the impression that the page is frozen and do not highlight the action popover.